web/src/settings_ui.ts

```diff
@@ -57,7 +57,7 @@ export function do_settings_change(
         success(response_data) {
             setTimeout(() => {
                 ui_report.success(success_msg_html, $spinner, remove_after);
-                display_checkmark($spinner);
+                // display_checkmark($spinner);
```

Comment:
Don't just comment out code; that leaves the codebase worse for future contributors. If the code is no longer correct, remove it.

Reply:
I'll make the necessary fixes and keep this in mind for future pull requests.